src/main/java/com/example/bookshop/model/User.java

```diff
@@ -39,4 +47,45 @@ public class User {
 
     @Column(nullable = false)
     private boolean isDeleted = false;
+
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @ManyToMany(fetch = FetchType.LAZY)
```
And handle LazyInitializationException